### ISSUE TITLE
NullPointerException in javadocjavaprint when @link points to not existing property

### ISSUE THREAD

**Issue by Postremus** [openrewrite/rewrite#1048]
I have following class definition:
I have a visitor reacting on my Allow Annotation.
However, I get the stacktrace from below when I execute my recipe.

The #SCHEDULER_MAX_EXECUTION_TIME does not exist in the class.
```
package com.company;
            
            import com.company.Allow;
            import com.company.Right;
            
            class A {
            /**
             * Tries to import stuff.<br/>
             * Will run for {@link #SCHEDULER_MAX_EXECUTION_TIME} ms
             */
@Allow(Right.SomeRight)
            public void read() {
            }
            }
```

Stacktrace
```
java.lang.NullPointerException
	at org.openrewrite.java.tree.J$MemberReference.getContaining(J.java:2745)
	at org.openrewrite.java.JavadocPrinter$JavadocJavaPrinter.visitMemberReference(JavadocPrinter.java:425)
	at org.openrewrite.java.tree.J$MemberReference.acceptJava(J.java:2794)
	at org.openrewrite.java.tree.J.accept(J.java:59)
	at org.openrewrite.TreeVisitor.visit(TreeVisitor.java:171)
	at org.openrewrite.java.JavadocPrinter.visitLink(JavadocPrinter.java:202)
	at org.openrewrite.java.tree.Javadoc$Link.acceptJavadoc(Javadoc.java:329)
	at org.openrewrite.java.tree.Javadoc.accept(Javadoc.java:38)
	at org.openrewrite.TreeVisitor.visit(TreeVisitor.java:171)
	at org.openrewrite.java.JavadocPrinter.visit(JavadocPrinter.java:62)
	at org.openrewrite.java.JavadocPrinter.visit(JavadocPrinter.java:383)
	at org.openrewrite.java.JavadocPrinter.visitDocComment(JavadocPrinter.java:107)
	at org.openrewrite.java.tree.Javadoc$DocComment.acceptJavadoc(Javadoc.java:161)
	at org.openrewrite.java.tree.Javadoc.accept(Javadoc.java:38)
	at org.openrewrite.TreeVisitor.visit(TreeVisitor.java:171)
	at org.openrewrite.java.JavadocPrinter.visit(JavadocPrinter.java:62)
	at org.openrewrite.java.JavadocPrinter.print(JavadocPrinter.java:49)
	at org.openrewrite.java.tree.Javadoc.print(Javadoc.java:52)
	at org.openrewrite.Tree.print(Tree.java:71)
	at org.openrewrite.java.tree.Javadoc$DocComment.printComment(Javadoc.java:166)
	at org.openrewrite.java.JavaPrinter.visitSpace(JavaPrinter.java:112)
	at org.openrewrite.java.JavaPrinter.visitMethodDeclaration(JavaPrinter.java:782)
	at org.openrewrite.java.tree.J$MethodDeclaration.acceptJava(J.java:2986)
	at org.openrewrite.java.tree.J.accept(J.java:59)
	at org.openrewrite.TreeVisitor.visit(TreeVisitor.java:171)
	at org.openrewrite.java.JavaPrinter.visit(JavaPrinter.java:67)
	at org.openrewrite.java.JavaPrinter.visitStatement(JavaPrinter.java:424)
	at org.openrewrite.java.JavaPrinter.visitStatements(JavaPrinter.java:415)
	at org.openrewrite.java.JavaPrinter.visitBlock(JavaPrinter.java:407)
	at org.openrewrite.java.tree.J$Block.acceptJava(J.java:678)
	at org.openrewrite.java.tree.J.accept(J.java:59)
	at org.openrewrite.TreeVisitor.visit(TreeVisitor.java:171)
	at org.openrewrite.java.JavaPrinter.visit(JavaPrinter.java:67)
	at org.openrewrite.java.JavaPrinter.visitClassDeclaration(JavaPrinter.java:531)
	at org.openrewrite.java.tree.J$ClassDeclaration.acceptJava(J.java:941)
	at org.openrewrite.java.tree.J.accept(J.java:59)
	at org.openrewrite.TreeVisitor.visit(TreeVisitor.java:171)
	at org.openrewrite.java.JavaPrinter.visit(JavaPrinter.java:67)
	at org.openrewrite.java.JavaPrinter.visit(JavaPrinter.java:77)
	at org.openrewrite.java.JavaPrinter.visitCompilationUnit(JavaPrinter.java:545)
	at org.openrewrite.java.tree.J$CompilationUnit.acceptJava(J.java:1153)
	at org.openrewrite.java.tree.J.accept(J.java:59)
	at org.openrewrite.TreeVisitor.visit(TreeVisitor.java:171)
	at org.openrewrite.java.JavaPrinter.visit(JavaPrinter.java:67)
	at org.openrewrite.java.JavaPrinter.print(JavaPrinter.java:54)
	at org.openrewrite.java.tree.J.print(J.java:73)
	at org.openrewrite.RecipeScheduler.scheduleRun(RecipeScheduler.java:101)
	at org.openrewrite.Recipe.run(Recipe.java:255)
	at org.openrewrite.RecipeTest$DefaultImpls.assertChangedBase(RecipeTest.kt:113)
	at org.openrewrite.RecipeTest$DefaultImpls.assertChangedBase(RecipeTest.kt:64)
	at org.openrewrite.java.JavaRecipeTest$DefaultImpls.assertChanged(JavaRecipeTest.kt:65)
	at com.company.Test.assertChanged(GenerateWolfAuthorizationRecipeTest.kt:7)
	at org.openrewrite.java.JavaRecipeTest$DefaultImpls.assertChanged$default(JavaRecipeTest.kt:49)
	at com.company.Test.test3(GenerateWolfAuthorizationRecipeTest.kt:82)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:725)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:149)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:140)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:84)
	at org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(ExecutableInvoker.java:115)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:98)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$6(TestMethodTestDescriptor.java:210)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:206)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:131)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:65)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:107)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
	at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
	at com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs(JUnit5IdeaTestRunner.java:71)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:33)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)


Process finished with exit code -1
```

**Comment by traceyyoshima** [comment]
@Postremus Hi, thanks for opening an issue!
I think the NPE was caused by the `<br/>` and should be fixed on the latest head by #1049.

**Comment by traceyyoshima** [comment]
Thanks for reporting the issues, closing the ticket, since it's fixed by  #1049 and #1050

**Comment by Postremus** [comment]
@traceyyoshima Thank you for the fast fix. This now works in my test cases, thank you!
